src/Checker.js

```diff
@@ -30,14 +31,23 @@ class Checker {
 
     this._updateTaskDefs(serviceNames, serviceStates);
 
-    return {
-      serviceDeployStates,
+    const alerts = {
       serviceDeployingAlerts,
       serviceScalingAlerts,
       serviceRecoverAlerts,
       serviceDeployDoneAlerts,
       serviceDeployTimeoutAlerts,
     };
+
+    if (this._hasAlerts(alerts)) {
+      getLogger().debug('An alert was created', { ...serviceDeployStates });
```

Comment:
Would you see debug messages? We're not filtering them?

Reply:
For me they're on by default in DDG. But even if not, these logs are really just for debugging so you can just turn them on